src/converter.ts

```diff
@@ -253,6 +262,20 @@ export class Converter {
     return ret
   }
 
+  private convertFileToNotes(tpl: TemplateResult, file: File): File {
+    const ret = file.convert(this.options.output, { extension: 'txt' })
+    const comments = new Marp().render(file.buffer?.toString() || '').comments
```

Comment:
The rendered notes have already stored in `tpl.rendered.comments`. Please use `tpl.rendered.comments` instead of the rendered result from orphan Marp instance because it may bypass engine customization by user.